test/locater_test.js

```diff
@@ -56,8 +73,8 @@ describe("locater", function() {
     });
 
     it("matches multiple occurences of a regex in a line", function() {
-      var result = locater.find(/[a-zA-Z]{6}/g, 'kaskade - 4AM');
```

Comment:
Not sure if this test was behaving as expected as it was only matching once. [a-zA-Z]{6} -> kaskad

Let me know if I'm missing something with my change.